spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb

```diff
@@ -47,6 +47,39 @@
         end
       RUBY
     end
+
+    context 'with single-key hash argument' do
+      it 'reports an offence' do
+        expect_violation(<<-RUBY)
+          before do
+            allow(foo).to receive_message_chain(bar: 42)
+                          ^^^^^^^^^^^^^^^^^^^^^ Use `receive` instead of calling `receive_message_chain` with a single argument.
+          end
+        RUBY
+      end
+
+      include_examples(
+        'autocorrect',
+        'before { allow(foo).to receive_message_chain(bar: 42) }',
+        'before { allow(foo).to receive(:bar) { 42 } }'
```

Comment:
Actually I prefer the `.and_return`. I left the block one just for consistency with the other test cases. On the other hand `and_return` is closer to the original code (no lazy evaluation), so I will replace it with `and_return`